Description: Remove dependencies that may no be found at runtime
 Some distributions are not found by pkg_resources when load_entry_point is
 called at startup
Author: Nick Morrott <nickm@debian.org>
Forwarded: not-needed
Last-Update: 2020-06-15
---
--- a/setup.py
+++ b/setup.py
@@ -36,6 +36,9 @@
     'Flask>=1.0.2',
 ]
 
+exclude = ('PyQt5', 'QScintilla', 'qtconsole', 'PyQtChart')
+install_requires = [requirement for requirement in install_requires
+        if not requirement.startswith(exclude)]
 
 extras_require = {
     'tests': [
